src/main/resources/application.conf

```diff
@@ -49,7 +50,8 @@ constellation {
     }
   }
   health-check {
-    p2p-interval = 10s
+    p2p-interval = 5s
+    max-timespan-without-own-reconciliation-round = 6h
```

Comment:
Are these parameters you tested it on testnet with?

Reply:
Yeah I didn't inject the separate application.conf and I don't see dag script doing this so these are the values used on testnet as well.